deploy/crds/cloudingress.managed.openshift.io_apischemes_crd.yaml

```diff
@@ -1,38 +1,67 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: apischemes.cloudingress.managed.openshift.io
+  name: APISchemes.cloudingress.managed.openshift.io
```

Comment:
This is known, but just to document here: the CRD needs to be lower case, and fields beneath the Names block should match it as well.

```
The CustomResourceDefinition "APISchemes.cloudingress.managed.openshift.io" is invalid: 
* metadata.name: Invalid value: "APISchemes.cloudingress.managed.openshift.io": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character
```